test/integration/targets/etcd3/vars/RedHat.yml

```diff
@@ -0,0 +1,3 @@
+etcd3_deps_packages:
+  - gcc-c++
+  - python-devel
```

Comment:
FYI that passing in multiple packages was previously undocumented functionality for the `package` module; making a note in #34290.